mlpp_lib/datasets.py

```diff
@@ -155,20 +154,22 @@ def apply_filter(self):
 
     def standardize(self, stage=None):
```

Comment:
I guess we should rename this to `normalize()` despite the breaking change, what do you think?

Reply:
See the above comment, I understand the necessity to change but looking in the future, I would call it `transform()`, so that we don't have a new breaking change if we decide to introduce transformations as well as normalizations.